interp/src/interpreter/interpret_group.rs

```diff
@@ -196,300 +141,185 @@ fn get_done_port(group: &ir::Group) -> RRC<ir::Port> {
 }
 
 // XXX(Alex): Maybe rename to `eval_is_done`?
-fn grp_is_done(done: OutputValueRef) -> bool {
+fn signal_is_high(done: OutputValueRef) -> bool {
```

Comment:
Random naming nit: prefer `is_signal_high` since the method is "answering" the question with a bool?

Reply:
I miss the ability to include `?` in identifiers. Scheme had me spoiled in that respect